engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -102,9 +102,11 @@ class WorkflowManagerActor(config: Config)
 
   private val donePromise = Promise[Unit]()
 
+  private val conf = ConfigFactory.load
+
   override def preStart() {
     addShutdownHook()
-    restartIncompleteWorkflows()
+    if (conf.getBoolean("system.workflow-restart")) { restartIncompleteWorkflows() }
```

Comment:
I've always viewed the 'run' mode picking up old workflows as bug, not feature (and thus would agree w/ your suggestion) although in the back of my mind I remember someone making a strong argument the other way so I don't think that's a universally held opinion.